hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestSplitTransactionOnCluster.java

```diff
@@ -504,7 +504,7 @@ public void testShutdownFixupWhenDaughterHasSplit() throws Exception {
   private void clearReferences(HRegion region) throws IOException {
     // Presumption.
     assertEquals(1, region.getStores().size());
-    HStore store = region.getStores().get(0);
+    HStore store = new ArrayList<>(region.getStores()).get(0);
```

Comment:
minor: you can replace with Iterables.getOnlyElement() here

Reply:
let me change this.
also, in fact, we can use Iterables.get() or Iterables.getFirst() for the rest